lib/administrate/field/number.rb

```diff
@@ -30,6 +32,11 @@ def decimals
       def value
         data * options.fetch(:multiplier, 1)
       end
+
+      def add_delimiter(result)
+        result.reverse.gsub(/(\d{3})(?=\d)/, "\\1#{options[:delimiter]}").
```

Comment:
How about using `ActiveSupport::NumberHelper.number_to_delimited`?

Reply:
Thanks for the comment, @sedubois. Sure, that sounds good to me. Using `ActiveSupport::NumberHelper.number_to_delimited` will make it easier to understand at a glance as well.